src/game.cpp

```diff
@@ -176,13 +177,17 @@ void Game::init() {
 			throw CreateException("Quantum Break is not yet supported by OpenAWE.");
 		case kResultAlanWakeRemastered:
 			throw CreateException("Alan Wake Remastered is not yet supported by OpenAWE.");
+		case kResultControl:
+			throw CreateException("Control is not yet supported by OpenAWE.");
+			//prefix = "data/";
```

Comment:
In your current version of the code, `prefix` variable is always an empty string. Since this was the only assignment before commenting out, you should probably remove it. However, if Control does store fonts in a different folder, you could reflect that in a comment for future use.

Reply:
I checked this, control stores the font in in the same directory.

Reply:
@Nostritius , Control stores everything in the rdpm archive under a /data folder, for example /data/fonts/fixedsys.binfnt as opposed to /fonts/fixedsys.bin for games like Alan Wake AN. (For completeness, some rpdm files with textures use the data_pc/ folder).
This is the reason why this prefix variable was present (for future use / reminder). I will remove the variable for now, but can you doublecheck that your version also uses a /data folder (in the rpdm archive itself) since you were saying it uses the same directory?

Reply:
Ah I'm sorry, I see my mistake. You are right, control changed the folder layout again. In Alan Wake the top level directory already contained the folders, while American Nightmare and Quantum Break had a d:/data folder prefix which is removed in the archive loading to  to align the individual file systems more together, since they have inside this prefix the same structure. If we ever want to support Control fully we would need to rethink that approach, for now it might be enough to say that we do not support this and exit accordingly.

Reply:
I understand what you mean. I wasn't aware of the D;/data removal - I indeed see it in the loading code.
There is one extra reference to /data in my original commit in meshman.cpp. I will remove these changes.